src/blocks/shared/utils/api.ts

```diff
@@ -0,0 +1,34 @@
+export interface AnalyticsApiQueryParams extends AnalyticsApiOptionalQueryParams {
+	type: string,
+}
+
+export interface AnalyticsApiOptionalQueryParams extends ApiPeriodRange {
```

Comment:
I'm unsure we need `Optional` in the name?

Reply:
In Parsely API docs we have separate section for Optional params so just follows that and it is also making things clear just from the name.